### DISCUSSION TITLE
How to program pause

### DISCUSSION THREAD

**Discussion by ghost** [ScratchAddons/ScratchAddons#766] [Help] [answered]
How do I use sa-pause to add pause to my project?

**Comment by TheColaber** [comment]
create a block, call it sa-pause. very simple.

**Comment by WorldLanguages** [comment, marked as answer]
Create a custom block with that name. Make the custom block do nothing. Then, use the new `sa-pause` block.

**Reply by ghost** [reply]
How do I use it

**Reply by WorldLanguages** [reply]
Just drag and drop the block to the moment where you want the project to pause.
